src/index.html

```diff
@@ -6,9 +6,62 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    >
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img 
+            class="icon"
+            src="./images/logo.png"
+            alt="icon MOYO"
+          >
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#apple" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a href="#samsung" class="nav_link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="#smartphones" class="nav_link">Smartphones</a>
+          </li>
```

Comment:
add blank line between same level multiline elements
```suggestion
        <ul class="nav_list">
          <li class="nav_item">
            <a href="#apple" class="nav_link is-active">Apple</a>
          </li>
          
          <li class="nav_item">
            <a href="#samsung" class="nav_link">Samsung</a>
          </li>
          
          <li class="nav_item">
            <a href="#smartphones" class="nav_link">Smartphones</a>
          </li>
```